### ISSUE TITLE
PreferredCollectionMethods finds false positive when 'inject' is set to 'inject'

### ISSUE THREAD

**Issue by geoffharcourt** [rubocop/rubocop#894]
There seems to be a problem with the configuration for this cop.

From my .rubocop.yml:

```
CollectionMethods:
  Enabled: true
  PreferredMethods:
    inject: 'inject'
```

When I check my code, I get this error:

```
app/models/wmata_metro_stop_populator.rb:59:14: C: Prefer inject over inject.
      (1..4).inject([]) do |memo, number|
             ^^^^^^
```

Clearly I should prefer inject over inject, right?
